### PR TITLE
feature(oidc):auth sdk add UpdateUserPasswordAsync 、UpdateUserBaseInfoAsync，oidc add transaction commit

### PR DESCRIPTION
**Pull request by 15168440402**

### CODE REVIEW

File: src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -18,27 +20,16 @@ public static class ServiceCollectionExtensions
         return services;
     }
 
-    public static IServiceCollection SeedClientData(this IServiceCollection services, List<Client> clients)
+    public static async Task AddOidcDbContext<T>(this IServiceCollection services, Func<OidcDbContextOptions, Task> options) where T : DbContext
```
Comment (zhenlei520) [inline src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/ServiceCollectionExtensions.cs:23]: Func? Action?

Comment (zhenlei520) [reply src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/ServiceCollectionExtensions.cs:23]: where T : IMasaDbContext, DbContext

File: src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -18,27 +20,16 @@ public static class ServiceCollectionExtensions
...
-        }
-        foreach (var client in clients)
-        {
-            if (clientRepository.FindAsync(s => s.ClientId == client.ClientId).Result == null)
-            {
-                _ = clientRepository.AddAsync(client).Result;
-            }
-        }
-        return services;
+            await unitOfWork.CommitAsync();
```
Comment (zhenlei520) [inline src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/ServiceCollectionExtensions.cs:32]: unitOfWork?.CommitAsync()

File: src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/_Imports.cs [hunk 3]
```diff
@@ -13,3 +13,4 @@
 global using Microsoft.EntityFrameworkCore.Metadata.Builders;
 global using Microsoft.Extensions.DependencyInjection;
 global using System.Linq.Expressions;
+global using Masa.Contrib.Authentication.Oidc.EntityFrameworkCore.Options;
```
Comment (zhenlei520) [inline src/Authentication/Masa.Contrib.Authentication.Oidc.EntityFrameworkCore/_Imports.cs:16]: global sort

File: test/Masa.Contrib.BasicAbility.Auth.Tests/UserServiceTest.cs [hunk 4]
```diff
@@ -151,5 +151,39 @@ public async Task TestGetUserVisitedListAsync()
         callerProvider.Verify(provider => provider.GetAsync<object, List<UserVisitedModel>>(requestUri, It.IsAny<object>(), default), Times.Once);
         Assert.IsTrue(result is not null);
     }
+
+    [TestMethod]
+    public async Task TestUpdateUserPasswordAsync()
```
Comment (zhenlei520) [inline test/Masa.Contrib.BasicAbility.Auth.Tests/UserServiceTest.cs:156]: Should methods reflect the purpose of unit testing?

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**